lib/cc/analyzer/issue_validations/other_locations_format_validation.rb

```diff
@@ -25,7 +30,8 @@ def message
         private
 
         def other_location_valid?(location)
-          CHECKS.all? do |klass|
+          path = location && location["path"]
+          @location[path] ||= CHECKS.all? do |klass|
```

Comment:
Sorry, on second thought maybe `other_location_validities` would be better? I didn't realize this was caching the resulting boolean, not the locations themselves.